configs/wsproxy/halfstack.toml

```diff
@@ -1,12 +1,12 @@
 [wsproxy]
-bind_host = "0.0.0.0"
-advertised_host = "127.0.0.1"
+bind-host = "0.0.0.0"
+advertised-host = "127.0.0.1"
 
-bind_api_port = 5050
-advertised_api_port = 5050
+bind-api-port = 5050
+advertised-api-port = 5050
 
 # replace these values with your passphrase
-jwt_encrypt_key = "QRX/ZX2nwKKpuTSD3ZycPA"
-permit_hash_key = "gHNAohmntRV0t9zlwTVQeQ"
+jwt-encrypt-key = "QRX/ZX2nwKKpuTSD3ZycPA"
+permit-hash-key = "gHNAohmntRV0t9zlwTVQeQ"
 
-api_secret = "v625xZLOgbMHhl0s49VuqQ"
+api-secret = "v625xZLOgbMHhl0s49VuqQ"
```

Comment:
Do you need to change the config file?
I think you can make it backwards compatible.

Reply:
Backward compatibility!